File: problemas/ad-hoc/2373/2373.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI
{
    static void Main(string[] args)
    {
        int resposta = 0;

        int N = int.Parse(Console.ReadLine());
        for (int i = 0; i < N; ++i)
        {
            List<int> LC = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();
            int L = LC[0];
            int C = LC[1];

            if (L > C)
            {
                resposta += C;
            }
        }

        Console.WriteLine(resposta);
    }
}

File: problemas/ad-hoc/2373/2373.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let n: i32 = input.trim().parse().unwrap();

    let mut resposta = 0;
    for _ in 0..n {
        input = String::new();
        io::stdin().read_line(&mut input);
        let numeros: Vec<i32> = input.split_whitespace().map(|s| s.parse().expect("")).collect();

        if (numeros[0] > numeros[1]) {
            resposta += numeros[1];
        }
    }

    println!("{}", resposta);
}
